### ISSUE TITLE
"Folder Open" has different design on Google Fonts and Figma

### ISSUE THREAD

**Issue by kissge** [google/material-design-icons#1593]
**Google Fonts** https://fonts.google.com/icons?icon.query=folder+open

<img width="312" alt="image" src="https://github.com/google/material-design-icons/assets/4177010/a5e41e3c-d5a3-458e-b3bb-951ea3cbed92">

**Figma** https://www.figma.com/community/file/1014241558898418245

<img width="314" alt="image" src="https://github.com/google/material-design-icons/assets/4177010/5fbd4a95-d667-45c3-b974-4cee22246c47">

Which is correct? IMHO the one in Google Fonts looks better :)

**Comment by tphinney** [comment]
Yes, that is the correct icon for /folder_open — the other one is just wrong, it is a plain folder.

All the folder icons went through a change in recent months, an update to match the internal Google version of those icons, with the tab hollowed instead of filled.

At some point, possibly during that effort, /folder_open got swapped for the plain folder icon. Not just in the Figma version, but on Google Fonts. That eventually got spotted and fixed—I remember fixing it. But the Figma version has not yet gotten that update.

Some time in the next couple of weeks there is going to be a big update to the external version of the font. That would be an excellent time to also update the Figma plug-in.

**Comment by kissge** [comment]
Thanks for your prompt response! Love to see the Figma upgrade soon 😍

**Comment by tphinney** [comment]
@kissge did this get resolved? Please either let me know if it did not, or close the issue if it did?

**Comment by kissge** [comment]
Not at all. Nothing's changed. [You can visit Figma and see "Last updated 6 months ago."](https://www.figma.com/community/file/1014241558898418245/material-design-icons)

![image](https://github.com/google/material-design-icons/assets/4177010/b449ef40-0832-41a7-87f6-be4a4aba784d)

**Comment by banerjeesayantan** [comment]
Hi team, the issue is still present in the Figma Material Symbols plugin.
Google Fonts has the updated “folder_open” icon, but the Figma plugin continues to show the old incorrect version.

The plugin hasn’t been updated for months (“Last updated 24 months ago”), so the latest icon set hasn’t synced.
Could you please push the updated Material Symbols into the Figma plugin?

This will help maintain consistency between Google Fonts and Figma.
Thanks for your work!

**Comment by tphinney** [comment]
It is a LOT older than that. The icons posted there date back to before Google made Material Symbols available. The current set has not only many design differences/tweaks, but also more than half again as many icons.

I have started making some inquiries about this... not sure whose responsibility the update would be. The person who posted it is not at Google any more.

**Comment by tphinney** [comment]
Ah, the Material Symbols icons are ALSO available in Figma!
https://www.figma.com/community/plugin/1088610476491668236/material-symbols

Still a couple of years old, but will have both more icons and newer and more consistent design choices.
